pkgs/by-name/kr/krename/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  fetchurl,
+  fetchFromGitLab,
+  lib,
+  extra-cmake-modules,
+  kdePackages,
+  taglib,
+  exiv2,
+  podofo,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krename";
+  version = "5.0.60-unstable-2025-09-02";
+
+  # For when the next stable release is made
+  # src = fetchurl {
+  #   url = "mirror://kde/stable/krename/${version}/src/krename-${finalAttrs.version}.tar.xz";
+  #   hash = "sha256-sjxgp93Z9ttN1/VaxV/MqKVY+miq+PpcuJ4er2kvI+0=";
+  # };
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "utilities";
+    repo = "krename";
+
+    rev = "5ad5f5a1f0a1c7573fa1b872a1472dec96fe0dd7";
+    hash = "sha256-fGNiIGGq10F71wh37aKDB7Q3fCxSXqttg/176LH3nVM=";
+  };
+
+  buildInputs = with kdePackages; [
+    exiv2
+    podofo
```

Krename doesn't detect podofo in 1.0 but only in 0.10 (tested)
```suggestion
  podofo_0_10,
}:

stdenv.mkDerivation (finalAttrs: {
  pname = "krename";
  version = "5.0.60-unstable-2025-09-02";

  # For when the next stable release is made
  # src = fetchurl {
  #   url = "mirror://kde/stable/krename/${version}/src/krename-${finalAttrs.version}.tar.xz";
  #   hash = "sha256-sjxgp93Z9ttN1/VaxV/MqKVY+miq+PpcuJ4er2kvI+0=";
  # };

  src = fetchFromGitLab {
    domain = "invent.kde.org";
    owner = "utilities";
    repo = "krename";

    rev = "5ad5f5a1f0a1c7573fa1b872a1472dec96fe0dd7";
    hash = "sha256-fGNiIGGq10F71wh37aKDB7Q3fCxSXqttg/176LH3nVM=";
  };

  buildInputs = with kdePackages; [
    exiv2
    podofo_0_10
```

---

Ah, it was an optional package and thus the build didn't fail. Fixed.